doc/user/content/free-trial-faqs.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Free trial FAQs"
+description: "Answers to frequency asked questions about Materialize free trials"
+menu:
+  main:
+    parent: "quickstarts"
+    weight: 15
+---
+
+When you [sign up for Materialize](https://materialize.com/register/), you get a free trial account so you can explore the product without worrying about initial costs. This page answers some frequently asked questions about free trials.
+
+## What's included in a free trial?
+
+In Materialize, [cluster replicas](/overview/key-concepts/#cluster-replicas) are the physical resources for doing computational work. Cluster replicas come in various sizes, and each size corresponds to a specific number of [compute credits](https://materialize.com/pricing/).
+
+During your free trial, you get up to 4 compute credits per region at any given time. This limit should accommodate most trial scenarios. If you need additional resources during your trial, please contact sales@materialize.com.
+
+## How long does a free trial last?
+
+14 days.
+
+If you want to continue using Materialize beyond 14 days, please contact sales@materialize.com. Otherwise, Materialize will delete your resources and data at the end of the trial period.
+
+## How do I monitor credit usage?
+
+To see how many compute credits you are currently using, run the following query:
+
+```sql
+SELECT sum(s.credits_per_hour) AS total_credits_per_hour
+FROM mz_cluster_replicas r
+JOIN mz_internal.mz_cluster_replica_sizes s ON r.size = s.size;To
+```
+
+For example, if you start your free trial with the [getting started guide](/get-started/), you'll end up consuming `3.5` credits:
```

Comment:
```suggestion
For example, if you start your free trial with the [getting started guide](/get-started/), you'll end up consuming credits at a rate of `3.5` per hour:
```

Reply:
Thanks, @benesch. I think I'll also rename the alias for the `SELECT` output from `total_credits_per_hour` to `credit_consumption_rate`.